teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -327,139 +246,213 @@ $ kubectl describe task.tekton.dev/golang-lint
-
 ```yaml
-# resource from the openshift catalog
+# annotations on resource from the "buildpacks" community catalog
+annotations:
+  tekton.dev/pipelines.minVersion: "0.7.0"
+  tekton.dev/tags: build
+  tekton.dev/catalog: buildpacks.tekton-integration
+  tekton.dev/catalog-support-tier: community
+  tekton.dev/catalog-url: https://github.com/buildpacks/tekton-integration
+
+# annotations on resource from the "openshift" community catalog
 annotations:
   tekton.dev/pipelines.minVersion: "0.17.0"
-  tekton.dev/tags: openshift-build
-  tekton.dev/catalog: com.redhat.openshift
+  tekton.dev/tags: build
+  tekton.dev/catalog: openshift.catalog
+  tekton.dev/catalog-support-tier: community  
   tekton.dev/catalog-url: https://github.com/openshift/catalog
 
-# resource from the gke catalog
+# annotations on resource from the "git" verified catalog
 annotations:
   tekton.dev/pipelines.minVersion: "0.30.0"
-  tekton.dev/tags: gke-build
-  tekton.dev/catalog: com.google.gke
-  tekton.dev/catalog-url: https://github.com/gke/catalog
-```
+  tekton.dev/tags: gitops
+  tekton.dev/catalog: tektoncd.git
+  tekton.dev/catalog-support-tier: verified
+  tekton.dev/catalog-url: https://github.com/tektoncd-catalog/git
 
-#### Community and Official Catalogs in Tekton Bundles
+# annotations on resource from the "kaniko" verified catalog
+annotations:
+  tekton.dev/pipelines.minVersion: "0.30.0"
+  tekton.dev/tags: build
+  tekton.dev/catalog: tektoncd.kaniko
+  tekton.dev/catalog-support-tier: verified
+  tekton.dev/catalog-url: https://github.com/tektoncd-catalog/kaniko
+```
 
-We propose publishing the resources in Catalogs as bundles separate OCI registries:
-* Community Catalog - `gcr.io/tekton-releases/catalog/upstream/<resource-name>:<resource-version>`
-* Official Catalog - `gcr.io/tekton-releases/catalog-official/upstream/<resource-name>:<resource-version>`
+#### Bundles
 
-Users can use the applicable reference to fetch and use the resource, as such:
+We propose publishing Tekton Bundles of the resources in the Verified Catalogs; note that we publish one Bundle per 
+resource currently. Users can use the applicable reference to fetch and use the resource, as such:
 
 ```yaml
-# bundle from a resource in the community catalog
+# bundle from a resource in the verified "git" catalog
 taskRef:
-    name: golang-lint
-    bundle: gcr.io/tekton-releases/catalog/upstream/golang-lint:0.1
+    name: git-clone
+    bundle: gcr.io/tekton-releases/catalogs/git/git-clone:0.7
 
-# bundle from a resource in the official catalog
+# bundle from a resource in the verified "kaniko" catalog
 taskRef:
-  name: golang-fuzz
-  bundle: gcr.io/tekton-releases/catalog-official/upstream/golang-fuzz:0.1
+  name: kaniko
+  bundle: gcr.io/tekton-releases/catalogs/kaniko/kaniko:0.3
 ```
 
-#### Community and Official Catalogs in Remote Resolution
+Contributors can publish Tekton Bundles from their own Catalogs using the resources provided by Tekton e.g. there
```

Comment:
are we allowing community users to publish their bundles on registry owned by Tekton? If not then should we be specific here that community catalogs can be published on any of the public registries which the OWNERS might have access to?

Reply:
Good point! I think community contributors should use their own registry. Update this section and PTAL 🙏